can-view-callbacks.js

```diff
@@ -259,18 +257,23 @@ var callbacks = {
 	attrs: attrs,
 	// handles calling back a tag callback
 	tagHandler: function(el, tagName, tagData){
+		// skip elements that have already been rendered
+		if (renderedElements.has(el)) {
+			return;
+		}
+
 		var scope = tagData.scope,
 			helperTagCallback = scope && scope.templateContext.tags.get(tagName),
 			tagCallback = helperTagCallback || tags[tagName],
 			res;
 
 		// If this was an element like <foo-bar> that doesn't have a component, just render its content
 		if(tagCallback) {
-			res = ObservationRecorder.ignore(tagCallback)(el, tagData);
-
 			// add the element to the Set of elements that have had their handlers called
 			// this will prevent the handler from being called again when the element is inserted
 			renderedElements.set(el, true);
+
+			res = ObservationRecorder.ignore(tagCallback)(el, tagData);
```

Comment:
Why does this need to be called after adding it to the Map?

Reply:
Ok, I ask because the test doesn't fail in Edge when this is reversed. Maybe another test case is needed?

Reply:
Actually I think this could probably be reverted back to normal now that the *skip elements that have already been rendered* has been centralised into the `tagHandler` function. Previously the skipping was being done in specific places and there were places that wouldn't be checking.